apis/workload/v1alpha1/types.go

```diff
@@ -95,8 +95,8 @@ type KubernetesApplicationStatus struct {
 	// State of the application.
 	State KubernetesApplicationState `json:"state,omitempty"`
 
-	// Cluster to which this application has been scheduled.
-	Cluster *KubernetesClusterReference `json:"clusterRef,omitempty"`
+	// Target to which this application has been scheduled.
+	Target *KubernetesTargetReference `json:"targetRef,omitempty"`
```

Comment:
I feel like we talked about this but I'm not sure. Just ignore my comment if that's already been decided.

Reply:
I tend to agree with you here @muvaf. This is not necessarily in-scope for this work (i.e. the change is not required for this new functionality), but if we are modifying the API anyway it might be good to go ahead and do this. Any thoughts @negz?

Reply:
https://github.com/crossplaneio/crossplane/pull/1146#discussion_r367061340

Thoughts here; I suppose I lean slightly toward moving this into spec.